test/azure/AcceptanceTests.cs

```diff
@@ -659,75 +657,5 @@ public void DurationTests()
             }
         }
 
-        [Fact]
-        public void ParameterGroupingTests()

```

Comment:
huh, why did you remove these?

Reply:
@olydis just pushed a change undoing this (which should fail on the CI), and I run into compilation issues, since you were working on this, can you take a look?